index.html

```diff
@@ -47,7 +47,7 @@
         <p class="ug-info triangle-border left">
           ReactJS.berlin is a usergroup focused on ReactJS and related
      topics. We meet regularly on the 2nd Monday each month at 7p.m.
-          <a href="http://bitcrowd.net" title="bitcrowd.net Office">@bitcrowd Office</a>,
+          <a href="http://bitcrowd.net" title="bitcrowd.net office">@bitcrowd Office</a>,
```

Comment:
Should we make the capitalisation consistent? The visible text is still '**O**ffice'